blog.md

```diff
@@ -14,7 +18,18 @@ their relationship with each other and with the remote repository.
 It finds an outstanding application in maintaining linear Git history.
 
 
-### Git Machete Status Graph
+## Git Machete
+
+The Git Machete plugin is a tool that helps developers handle multiple branches.
+The main features of the plugin are:
```

Comment:
I wouldn't call the below `features`, rather more high-level `objectives`? `aims`?...

Reply:
Also, `s/are/include` sounds better